resources/frameworks.yaml

```diff
@@ -203,6 +203,106 @@ AutoGluonTS:
   setup_env:
     MODULE: timeseries
 
+GluonTS:
```

Comment:
Add comment in this section that these frameworks are all baselines and not AutoML systems.